assets/oag.css

```diff
@@ -230,6 +238,54 @@
 		border: 1px solid transparent;
 		background-color: #F4F4F2;
 	}
+
+	.wb-tabs.vertical {
+		/* Vertical tab styling */
+		display: flex;
+		height: 100%;
+		[role="tablist"] {
+			width: max-content;
+			max-width: 30%;
+			display: block;
+			li {
+				display: block;
+				padding: 0;
+				text-align: left;				
+				border-top: 1px solid #959595;
```

Comment:
I don't see the border top being used. Is it always overwritten? Couldn't this be replaced by simply adding "border: none;" and then "border-bottom: 1px solid #959595" ?